livre/eq/Channel.cpp

```diff
@@ -275,16 +275,9 @@ class Channel
         _renderViewPtr->setViewport( viewport );
 
         livre::Window* window = static_cast< livre::Window* >( _channel->getWindow( ));
```

Comment:
DashTree is"not" always updated. This part of the code is critical. If you can test with synchronous mode, fivox, playing with animation I will be glad.